src/main/resources/assets/embeddiumplus/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+  "embeddium.plus.options.displayfps.title": "显示FPS",
+  "embeddium.plus.options.displayfps.desc": "显示当前FPS. 高级模式还显示最低FPS以及15秒的平均FPS，这对判断性能很有用。",
+  "embeddium.plus.options.displayfps.position.title": "FPS显示位置",
+  "embeddium.plus.options.displayfps.position.desc": "将FPS显示偏移几个像素",
+  "embeddium.plus.options.darkness.title": "真正的黑暗",
+  "embeddium.plus.options.darkness.desc": "使世界其他地方更加逼真地黑暗。不影响日间照明或火把照明。",
+  "embeddium.plus.options.darkness.mode.title": "真实黑暗方式",
+  "embeddium.plus.options.darkness.mode.desc": "控制什么样的黑暗被视为真正的黑暗。",
+  "embeddium.plus.options.darkness.mode.pitchblack": "漆黑一片",
+  "embeddium.plus.options.darkness.mode.reallydark": "真实黑暗",
+  "embeddium.plus.options.darkness.mode.dark": "黑暗",
+  "embeddium.plus.options.darkness.mode.dim": "昏暗",
+  "embeddium.plus.options.chunkfadeinquality.title": "区块渐变质量",
+  "embeddium.plus.options.chunkfadeinquality.desc": "控制区块消退的速度。没有性能消耗，Fancy只需要更长的时间，但看起来有点酷。当前不工作",
+  "embeddium.plus.options.fog.title": "启用雾",
+  "embeddium.plus.options.fog.desc": "关闭整个世界中的所有雾",
+  "embeddium.plus.options.jei.title": "在搜索之前隐藏JEI",
+  "embeddium.plus.options.jei.desc": "关闭JEI项目，除非您搜索某些内容。按空格键可搜索所有内容。",
+  "embeddium.plus.options.clouds.height.title": "云高度",
+  "embeddium.plus.options.clouds.height.desc": "增加云的高度。",
+  "embeddium.plus.options.culling.entity.title": "启用最大实体距离",
+  "embeddium.plus.options.culling.entity.desc": "关闭实体剔除。",
+  "embeddium.plus.options.culling.entity.distance.h.title": "实体最大距离(水平)",
+  "embeddium.plus.options.culling.entity.distance.h.desc": "隐藏并且不更新超过此格的实体。大幅提高性能，尤其是在刷怪塔周围。",
+  "embeddium.plus.options.culling.entity.distance.v.title": "实体最大距离(垂直)",
+  "embeddium.plus.options.culling.entity.distance.v.desc": "隐藏并且不更新这么多格下面的实体，提高洞穴上方的性能。理想情况下，该距离应设置为低于水平距离。",
+  "embeddium.plus.options.culling.tile.distance.h.title": "方块实体最大距离(水平)",
+  "embeddium.plus.options.culling.tile.distance.h.desc": "隐藏超过此格的方块实体。大幅提高性能，尤其是在许多改装机器周围。",
+  "embeddium.plus.options.culling.tile.distance.v.title": "方块实体最大距离(垂直)",
+  "embeddium.plus.options.culling.tile.distance.v.desc": "将在这么多格的下面方块实体隐藏，提高洞穴上方的性能 (如果出于某种原因，你把机器放在洞穴里). 理想情况下，该距离应设置为低于水平距离。",
+  "embeddium.plus.options.screen.title": "全屏模式",
+  "embeddium.plus.options.screen.desc": "窗口化-游戏将显示在一个小窗口中。\n无边界-游戏将被全屏显示，并锁定到显示器的刷新率，但允许您轻松退出。\n全屏-游戏将以本机全屏模式显示。",
+  "embeddium.plus.options.screen.windowed": "窗口化",
+  "embeddium.plus.options.screen.borderless": "无边框",
+  "embeddium.plus.options.dynlights.group": "动态光源",
+  "embeddium.plus.options.dynlights.speed.title": "光源更新速度",
+  "embeddium.plus.options.dynlights.speed.desc": "控制动态光源的更新频率。\n\n重新计算照明可能会带来更高的性能消耗，因此较慢的值会带来更好的性能。",
+  "embeddium.plus.options.dynlights.entities.title": "为实体启用",
+  "embeddium.plus.options.dynlights.entities.desc": "启用此选项将在实体上显示动态照明 (生物、掉落的物品等).\n\n即使你没有拿着火把，这也会大大增加照明更新的数量。",
+  "embeddium.plus.options.dynlights.blockentities.title": "为方块实体启用",
+  "embeddium.plus.options.dynlights.blockentities.desc": "启用此选项将在方块实体上显示动态照明 (熔炉、改装的机器等).\n\n即使你没有拿着火把，这也会大大增加照明更新的数量。"

```

顺便把这俩单位也翻译了。
By the way, these two units have also been translated.
```suggestion
  "embeddium.plus.options.dynlights.blockentities.desc": "启用此选项将在方块实体上显示动态照明 (熔炉、改装的机器等).\n\n即使你没有拿着火把，这也会大大增加照明更新的数量。",
  "Pixels": "像素",
  "Blocks": "格"

```

---

感谢你的更正。这是我第一次尝试本地化，我还有很多东西要学
Thank you for the correction. This is my first attempt at localization, and I still have a lot to learn